README.md

```diff
@@ -17,7 +17,8 @@ To learn how to contribute to the MetaMask project itself, visit our [Internal D
 - Install [Node.js](https://nodejs.org) version 16
   - If you are using [nvm](https://github.com/nvm-sh/nvm#installing-and-updating) (recommended) running `nvm use` will automatically choose the right node version for you.
 - Install [Yarn v3](https://yarnpkg.com/getting-started/install)
-- Install dependencies: `yarn`
+    - ONLY follow the steps in the "Install Corepack" and "Updating the global Yarn version" sections
+    - DO NOT take any of the steps in the "Initializing your project", "Updating to the latest versions" or "Installing the latest build fresh from master" sections. These steps could result in your repo being reset or installing the wrong yarn version, which can break your build.
```

Comment:
This might be easier to understand:
```suggestion
    - DO NOT take any of the steps in the "Initializing your project" section or later sections. These steps could result in your repo being reset or installing the wrong yarn version, which can break your build.
```